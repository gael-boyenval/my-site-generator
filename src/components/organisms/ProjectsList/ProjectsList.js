import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';

const X = styled.div``;

class ProjectsList extends PureComponent {
  static propTypes = {

  };

  static defaultProps = {

  };

  state = {

  };

  render() {
    const { } = this.props;
    const { } = this.state;

    return (
      <X> hello </X>
    );
  }
}

export default ProjectsList;
