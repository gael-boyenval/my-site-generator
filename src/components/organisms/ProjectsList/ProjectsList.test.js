/* eslint-disable no-undef */
import React from 'react';
import ProjectsList from './ProjectsList';

describe('Component ProjectsList', () => {
  it('should be render without crash', () => {
    const component = shallow(<ProjectsList />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
