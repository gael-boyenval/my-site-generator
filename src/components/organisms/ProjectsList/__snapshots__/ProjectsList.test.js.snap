// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component ProjectsList should be render without crash 1`] = `
<styled.div>
   hello 
</styled.div>
`;
